asterisk_click2dial/asterisk_click2dial.py

```diff
@@ -170,7 +170,7 @@ def _reformat_number(
         assert(country_code), 'Missing country on company'
         _logger.debug('Country code = %s' % country_code)
         to_dial_number = phonenumbers.format_out_of_country_calling_number(
-            parsed_num, country_code.upper()).replace(' ', '').replace('-', '')
+            parsed_num, country_code.upper()).replace(' ', '').replace('-', '').replace('.','').replace('(', '').replace(')','')
```

Comment:
@alex2grad regexp is quite slow and for stripping compared to translate
You must not strip `+` sign as it is valid if it is the first character